### ISSUE TITLE
Percent sign does not expand out to current file

### ISSUE THREAD

**Issue by jtw023** [folke/todo-comments.nvim#110]
Running the command `:TodoQuickFix cwd=%` produces the following output:
~~~
Todo: rg failed with code 2
/absolute/path/to/file/%: No such file or directory (os error 2)
~~~
I've also tried `:TodoQuickFix cwd=expand("%:p")` and `:TodoQuickFix cwd=expand("%")`. Those fail with the same output.

I've only been successful when adding the filepath manually.

`:TodoTelescope` also doesn't allow a cwd argument. The command `:TodoTelescope cwd=path/to/file` fails with output captured in the below screenshot. Is this expected?
![2022-05-27_23-35](https://user-images.githubusercontent.com/59935328/170813640-918baa57-7564-4979-bff0-030ef3579cc1.png)

**Comment by d11638104** [comment]
I have the same problem

**Comment by d11638104** [comment]
You may not have rg

**Comment by jtw023** [comment]
Solved it! I definitely have rg installed because I use it as my grep replacement outside of vim. What I ended up having to do is create a shortcut that runs the following command:
`:exe ":TodoQuickFix cwd=" .. fnameescape(expand("%:p"))<CR>`

To explain:
`:TodoQuickFix cwd=` is the command that I want to run and that is put inside of double quotes so that it gets interpreted by the execute command(`:exe`) as a string.

`expand("%:p")` is to print out the current file path just like running `:echo expand("%:p")`.

`fnameescape()` is to escape any spaces that may be returned in the expanded file path.

`..` is to concatenate ":TodoQuickFix cwd=" with the string returned from fnameescape.

That gives us `:TodoQuickFix cwd=/full/path/to/current/file` which is then executed as a vim command.
